### ISSUE TITLE
Can support ubuntu with npu?

### ISSUE THREAD

**Issue by dockerg** [intel-analytics/ipex-llm#12524]
I try run this project with ubuntu,but some dependence cant find(bigdl-core-npu,pipeline.so);
How can we run it with ubuntu?

**Comment by Oscilloscope98** [comment]
Hi @dockerg,

IPEX-LLM currently only supports Intel NPU on Windows. You could refer to [IPEX-LLM NPU QuickStart](https://github.com/intel-analytics/ipex-llm/blob/main/docs/mddocs/Quickstart/npu_quickstart.md) for more information :)

**Comment by dockerg** [comment]
Is there any plan for support on linux?

**Comment by dockerg** [comment]
> Hi @dockerg,
> 
> IPEX-LLM currently only supports Intel NPU on Windows. You could refer to [IPEX-LLM NPU QuickStart](https://github.com/intel-analytics/ipex-llm/blob/main/docs/mddocs/Quickstart/npu_quickstart.md) for more information :)

Is there any plan for support on linux?

**Comment by dockerg** [comment]
> Hi @dockerg,
> 
> IPEX-LLM currently only supports Intel NPU on Windows. You could refer to [IPEX-LLM NPU QuickStart](https://github.com/intel-analytics/ipex-llm/blob/main/docs/mddocs/Quickstart/npu_quickstart.md) for more information :)

Is there any plan for support on linux? @Oscilloscope98

**Comment by Oscilloscope98** [comment]
Hi @dockerg,

We currently do not have plans to support Linux on Intel NPU. We will share here for any updates :)

**Comment by Kepontry** [comment]
Agree, I'm also looking forward to the linux version. Are there any updates on the plan?

**Comment by wdkwyf** [comment]
@Oscilloscope98 Any update about Linux NPU support？

**Comment by amitbar05** [comment]
Yes ,please add support. I bought core ultra 7 265K because I was sure I could use the NPU on Ubuntu.
